packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -69,9 +71,9 @@ function Popover(props: PopoverProps, ref: DOMRef<HTMLDivElement>) {
 
 const PopoverWrapper = forwardRef((props: PopoverWrapperProps, ref: RefObject<HTMLDivElement>) => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  let {children, placement = 'bottom', arrowProps, isOpen, hideArrow, shouldCloseOnBlur, isKeyboardDismissDisabled, ...otherProps} = props;
+  let {children, placement = 'bottom', arrowProps, isOpen, hideArrow, shouldCloseOnBlur, isKeyboardDismissDisabled, isModal = true, ...otherProps} = props;
   let {overlayProps} = useOverlay({...props, isDismissable: true}, ref);
-  let {modalProps} = useModal();
+  let {modalProps} = isModal ? useModal() : {modalProps: {}} ;
```

Comment:
Unfortunately, due to the [rules of hooks](https://reactjs.org/docs/hooks-rules.html), conditional calls to hooks are not allowed. The option would need to be passed into `useModal` and checked [here](https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/overlays/src/useModal.tsx#L145) as well as [here](https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/overlays/src/useModal.tsx#L161).

Reply:
This is pretty much a hack right now, but I can make the change to follow the rules.